api/src/pcapi/routes/backoffice_v3/templates/titelive/search_result.html

```diff
@@ -0,0 +1,105 @@
+{% from "components/turbo/lazy_modal.html" import build_lazy_modal with context %}
+{% from "components/search/result_form.html" import build_result_page_form %}
+{% extends "layouts/connected.html" %}
+{% block page %}
+  <div class="pt-3 px-5">
+    <h2 class="fw-light">Recherche EAN via Tite live</h2>
```

Comment:
```suggestion
    <h2 class="fw-light">Recherche EAN via Tite Live</h2>
```
Et regarder partout pour être sûr que c'est bien `Tite Live` ;)